src/services/pcn-bridge/src/Fdb.cpp

```diff
@@ -18,7 +18,7 @@
 #include "Bridge.h"
 
 Fdb::Fdb(Bridge &parent, const FdbJsonObject &conf) : FdbBase(parent) {
-  logger()->info("Creating Fdb instance");
+  logger()->debug("[Fdb] Creating instance");
```

Comment:
BTW do we support logging based on sub-category like enable only "Fdb"?

Reply:
It would be nice, but the current logging infrastructure doesn't support this for now.